assignment08/index.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+    include "includes/header.php";
+    $my_weight= '125 lb';
```

Comment:
Ideally - this value would just be the  number. Then, we don't have to worry about what happens to the lb label or type conversion.